app/src/main/res/drawable/button_rounded_green.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android" android:shape="rectangle">
```

Comment:
Both myself and Katja have left comments about the buttons. I think you can add a theme to style the standard android buttons

Reply:
Done

Reply:
Thanks